package.json

```
@@ -0,0 +1,102 @@
+{
+  "name": "track-mail-2019-max",
+  "homepage": "https://your_gh_login.github.io/your_repo_name",
```

Понимаешь, почему здесь такой homepage должен быть? :)
```suggestion
  "homepage": "https://maxsos.github.io/2019-2-Track-Frontend-M-Koliev",
```